PWGLF/DataModel/LFStrangenessTables.h

```diff
@@ -165,6 +165,21 @@ DECLARE_SOA_TABLE(DauTrackMCIds, "AOD", "DAUTRACKMCID", // index table when usin
 using DauTrackExtras = DauTrackExtras_001;
 using DauTrackExtra = DauTrackExtras::iterator;
 
+DECLARE_SOA_TABLE(StoredDauTrackExtras, "AOD1", "DAUTRACKEXTRA", //! detector properties of decay daughters
```

Comment:
Is this used somewhere? I could not find it anywhere

Reply:
The "stored" versions of tables are necessary for the JE derived data format.
Although `DauTrackExtras` is not necessary for my specific analysis, I will likely be of importance to a future analysis.

Reply:
You could move it to the JE part then?

Reply:
I'm not sure that would work. @nzardosh would know, I think.

Reply:
Hi Nicolo, thanks for taking a look! At the moment it isn't used and we can remove it. But talking to David he said that this table is needed for analyses at later stages so we preemptively made it available. If you prefer though I'm happy to remove it for now. 

I would however prefer to keep them with the original table definitions since if you change the original table you need to correspondingly change these too, otherwise it won't compile (for the ones that are used). If it's in the JE framework it would make it difficult for the LF code writers to find it. 

Thanks,
Nima

Reply:
Ok, I did not have the full plan in mind